EIPS/eip-7441.md

```diff
@@ -0,0 +1,101 @@
+---
+eip: 7441
+title: Upgrade block proposer election to Whisk
+description: Allow elected block proposers to remain private until block publishing, to prevent DoS attacks
+author: George Kadianakis (@asn-d6), Justin Drake (@JustinDrake), dapplion (@dapplion)
+discussions-to: https://ethereum-magicians.org/t/eip-7441-upgrade-block-proposer-election-to-whisk-ssle/15316
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-09-01
+---
+
+## Abstract
+
+Upgrades the block proposer election mechanism to Whisk, a single secret leader election (SSLE) protocol. Currently, block proposers are publicly known in advance, sufficiently to allow sequential DoS attacks that could disable Ethereum. This upgrade makes the next block proposer unknown until its block is published.
+
+## Motivation
+
+The beacon chain currently elects the next 32 block proposers at the beginning of each epoch. The results of this election are public and everyone gets to learn the identity of those future block proposers.
+
+This information leak enables attackers to launch DoS attacks against each proposer sequentially in an attempt to disable Ethereum.
+
+## Specification
+
+### Execution layer
+
+This requires no changes to the Execution Layer.
+
+### Consensus layer
+
+The protocol can be summarized in the following concurrent steps:
+
+- Validators register a tracker and unique commitment on their first proposal after the fork
+- At the start of a shuffling phase a list of candidate trackers is selected using public randomness from RANDAO
+- During each shuffling phase each proposer shuffles a subset of the candidate trackers using private randomness
+- After each shuffling phase an ordered list of proposer trackers is selected from the candidate set using RANDAO
+
+The full specification of the proposed change can be found in [`/_features/whisk/beacon-chain.md`](https://github.com/ethereum/consensus-specs/blob/a39abe388bc2d1abd5b4fd62fd18aed497956b30/specs/_features/whisk/beacon-chain.md). In summary:
+
+- Update `BeaconState` with fields needed to track validator trackers, commitments, and the two rounds of candidate election.
```

Comment:
may be we don't need this detail as its defined in the `full specification` already refered and could do away with this

Reply:
Would be good define some guidelines on what this section is expected to include or not? Other editors can chip-in and @djrtwo